js/ccf-app/src/global.ts

```diff
@@ -251,6 +263,25 @@ export interface CCF {
    */
   isValidX509CertChain(chain: string, trusted: string): boolean;
 
+  crypto: {
+    /**
+     * Returns whether digital signature is valid.
```

Comment:
It would be good to list the RSA key sizes and EC curve names this supports for clarity, either here or in the definitions of the SigningAlgorithm interfaces.

Reply:
There is no restriction on the RSA key sizes or EC curves. Anything that OE-OpenSSL supports will work. The only limitation is in the supported digest algorithms which is already a TypeScript type and self-documented. I'd like to extend that from SHA-256 eventually to SHA-384 but want to do it across the board, so including ccf.digest(), and currently CCF doesn't have native SHA-384 digest support.
I found https://github.com/openenclave/openenclave/blob/master/docs/OpenSSLSupport.md but it's not that useful.

Reply:
Ok, thank you.